reactor-netty-http/src/test/java/reactor/netty/http/HttpMetricsHandlerTests.java

```diff
@@ -178,18 +178,18 @@ void testExistingEndpoint(HttpProtocol[] serverProtocols, HttpProtocol[] clientP
 
 		AtomicReference<SocketAddress> serverAddress = new AtomicReference<>();
 		httpClient = customizeClientOptions(httpClient, clientCtx, clientProtocols).doAfterRequest((req, conn) ->
-			serverAddress.set(conn.channel().remoteAddress())
+				serverAddress.set(conn.channel().remoteAddress())
 		).observe(observerDisconnect);
 
 		StepVerifier.create(httpClient.post()
-		                              .uri("/1")
-		                              .send(body)
-		                              .responseContent()
-		                              .aggregate()
-		                              .asString())
-		            .expectNext("Hello World!")
-		            .expectComplete()
-		            .verify(Duration.ofSeconds(30));
+						.uri("/1")
+						.send(body)
+						.responseContent()
+						.aggregate()
+						.asString())
+				.expectNext("Hello World!")
+				.expectComplete()
+				.verify(Duration.ofSeconds(30));
```

Comment:
Please revert the formatting changes in this class.

Reply:
Interesting, this seems to be either not in the .editorConfig, or not picked up by IntelliJ automatically.